### PR TITLE
ruby 2.3.1

### PR DESCRIPTION
**Pull request by chdiza**

### CODE REVIEW

File: Formula/ruby.rb [hunk 1]
```diff
@@ -3,18 +3,8 @@ class Ruby < Formula
   homepage "https://www.ruby-lang.org/"
 
   stable do
```
Comment (dunn) [inline Formula/ruby.rb]: You can move the `url` and `sha256` out of the stable block now.

**Comment by dunn** [comment]
Looks like it's still trying to install headers where it oughtn't:
```
installing common headers:    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/local/Cellar/ruby/2.3.1/include/ruby-2.3.0
/private/tmp/ruby-20160430-9880-1t2jffu/ruby-2.3.1/lib/fileutils.rb:250:in `mkdir': Operation not permitted @ dir_s_mkdir - /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/local (Errno::EPERM)
	from /private/tmp/ruby-20160430-9880-1t2jffu/ruby-2.3.1/lib/fileutils.rb:250:in `fu_mkdir'
	from /private/tmp/ruby-20160430-9880-1t2jffu/ruby-2.3.1/lib/fileutils.rb:227:in `block (2 levels) in mkdir_p'
	from /private/tmp/ruby-20160430-9880-1t2jffu/ruby-2.3.1/lib/fileutils.rb:225:in `reverse_each'
	from /private/tmp/ruby-20160430-9880-1t2jffu/ruby-2.3.1/lib/fileutils.rb:225:in `block in mkdir_p'
	from /private/tmp/ruby-20160430-9880-1t2jffu/ruby-2.3.1/lib/fileutils.rb:211:in `each'
	from /private/tmp/ruby-20160430-9880-1t2jffu/ruby-2.3.1/lib/fileutils.rb:211:in `mkdir_p'
	from ./tool/rbinstall.rb:191:in `makedirs'
	from ./tool/rbinstall.rb:304:in `prepare'
	from ./tool/rbinstall.rb:513:in `block in <main>'
	from ./tool/rbinstall.rb:801:in `block in <main>'
	from ./tool/rbinstall.rb:798:in `each'
	from ./tool/rbinstall.rb:798:in `<main>'

**Comment by chdiza** [comment]
I don't know what to do about that.  It built fine locally.

**Comment by chdiza** [comment]
Something weird is going on with git on my end.  I don't know how to solve it.

I'm going to close this PR and let someone who knows what they're doing bump the ruby version.

**Pull request closed without merging**